### ISSUE TITLE
Request: code_pipeline

### ISSUE THREAD

**Issue by nitrocode** [dtan4/terraforming#446]
Thank you for this useful cli app!

https://www.terraform.io/docs/providers/aws/r/codepipeline.html

**Comment by BartusZak** [comment]
Is there a possibility to export existing AWS CodePipeline configuration?

**Comment by nitrocode** [comment]
@BartusZak, yes because there is a way to import it using Terraform so the API supports it.

```
terraform import aws_codepipeline.foo example
```

See [available codepipeline commands](https://docs.aws.amazon.com/cli/latest/reference/codepipeline/index.html#available-commands)

**Comment by vibhuyadav** [comment]
+1
